control_panel_api/management/commands/migrate_lambdas_data_1_buckets.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import os
+
+import boto3
+from django.core.management.base import BaseCommand
+
+from control_panel_api.management.commands.migrate_lambdas_data_utils import (
+    bucket_name,
+    is_eligible,
+)
+from control_panel_api.models import S3Bucket
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    NOTE: Needs permission to perform `iam:ListPolicies` action
+    """
+
+
+    help = "Add records for S3 buckets created by AWS Lambda functions."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--dry-run',
```

Comment:
Just a suggestion - you could extract `add_arguments` into a mixin class

Reply:
Moved to `DryRunnable` 👍